pom.xml

```diff
@@ -1329,7 +1329,7 @@
                   <version>${java.version}</version>
                 </requireJavaVersion>
                 <requireMavenVersion>
-                  <version>3.3.9</version>
+                  <version>3.5.4</version>
```

Comment:
Please revert this change, if you have problems with this version please create separate issue to discuss it and if we approve issue we can do update, but let's do this in separate PR.

Reply:
done

Reply:
PR for maven version update - https://github.com/checkstyle/checkstyle/pull/13887